shed/event_streams.py

```diff
@@ -1256,13 +1286,14 @@ def update(self, x, who=None):
 
 
 class QueryUnpacker(EventStream):
-    def __init__(self, db, child, fill=True):
+    def __init__(self, db, child, fill=True, **kwargs):
```

Comment:
perhaps a docstring could be good here. I'm not familiar with this. Same with `Query`. maybe another PR?